qa/L0_tf_gpu_io/test.sh

```diff
@@ -104,21 +102,6 @@ for BACKEND in $BACKENDS; do
         RET=1
     fi
 
-    # Sample latency results
-    $CLIENT -m${MODEL_NAME}_def --shape INPUT0:${TENSOR_SIZE} \
-             >> ${BACKEND}.log 2>&1
-    if (( $? != 0 )); then
-        cat ${BACKEND}.log
-        RET=1
-    fi
-
-    $CLIENT -m${MODEL_NAME}_gpu --shape INPUT0:${TENSOR_SIZE} \
-            >> ${BACKEND}.gpu.log 2>&1
-    if (( $? != 0 )); then
-        cat ${BACKEND}.gpu.log
-        RET=1
-    fi
-
```

Comment:
Are these not needed?

Reply:
I don't believe so. We don't do anything with the latency results.

This seems to run perf_analyzer a second time to get the results and save them as a log. However, this seems to be redundant given that it's already run once, so I'd rather remove this second run to preserve CI resources (and the whole point of this ticket is to remove the need for perf_analyzer).